### PR TITLE
Add Time Plugin

### PR DESCRIPTION
**Pull request by jaxellis**
1. Add time.py, adds cmd .dt w/ flags [-l]
2. Uses COUNTRY_CITY var

PEP8

### CODE REVIEW

**Comment by Harsh-br0** [comment]
OwO
Pro guy(;_;)

File: plugins/time.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        " one in your Heroku Config Under</code> (<code>COUNTRY_CITY</code>)\n"
+        "<code>Ex: America/Los_Angeles</code>")
+
+    if 'l' in message.flags:
+        LOG.info("Time: List Flag Used: Giving TZ list...")
+        await message.edit(defaultMessage, disable_web_page_preview=True,
+                           parse_mode="html", del_in=30)
+        return
+
+    if not Config.COUNTRY_CITY:
```
Comment (rking32) [inline plugins/time.py:33]: should be `COUNTRY_CITY`

File: plugins/time.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        LOG.info("Time: No Config Set")
+        await message.edit(defaultMessage, disable_web_page_preview=True,
+                           parse_mode="html", del_in=30)
+        return
+
+    tz = Config.COUNTRY_CITY
```
Comment (rking32) [inline plugins/time.py:39]: here too

Comment (Phyco-Ninja) [reply plugins/time.py:39]: 🌚 remove import of Config also. Or Deepsource will give another suggestion🤔

Comment (jaxellis) [reply plugins/time.py]: @Phyco-Ninja yeah I caught that thanks :)

**Comment by rking32** [comment]
this worked right ?

**Comment by jaxellis** [comment]
> this worked right ?

https://i.imgur.com/nfWcNUW.png
https://i.imgur.com/PYEIjRH.png
https://i.imgur.com/xX4MZck.png
https://i.imgur.com/HHwIJxX.png

Yes :)

**Comment by rking32** [comment]
cool, merging ...

**Pull request merged**